src/lib/features/snackbar/ui/snackbar.svelte

```diff
@@ -0,0 +1,23 @@
+<script lang="ts">
+  import type { SnackbarOptions } from '../model/types';
+  import { filterQueueByText } from '../model/store';
+
+  export let snackbar: SnackbarOptions;
+
+  const callback = () => {
+    snackbar.callback();
+    filterQueueByText(snackbar.text);
+  };
+</script>
+
+<div
+  class="fixed bottom-16 mb-2 left-2 right-2 border border-white border-2 rounded-lg p-4 flex flex-col bg-background-slightly-higher"
```

Comment:
nitpick (non-blocking): `left-2 right-2` is not going to position it 8px from the left and right sides, it's gonna be one or the other. In this case it doesn't matter, I think